SunRise/Controllers/Library/LibraryCell/Favorite/FavoriteViewController.swift

```diff
@@ -0,0 +1,122 @@
+import UIKit
+
+final class FavoriteViewController: BaseViewController {
+    @IBOutlet weak private var tableView: UITableView!
+    @IBOutlet weak private var playButton: UIButton!
+    @IBOutlet weak private var favoriteButton: UIButton!
+    @IBOutlet weak private var titleLabel: UILabel!
+    @IBOutlet weak private var userTitle: UILabel!
+    
+    private let viewModel = FavoriteViewModel()
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+    }
+    
+    override func viewDidAppear(_ animated: Bool) {
+        updateData()
```

Comment:
thiếu super

Reply:
Dạ func này ko phải gọi tử base class anh Đạt!

Reply:
nó gọi tử `UIViewController` nên cần thêm super nhé

Reply:
Dạ em gán vào nó báo lỗi anh "Value of type 'BaseViewController' has no member 'updateData'"

Reply:
ý anh là thêm `super.viewDidAppear(animated)` ấy